src/journal/views.py

```diff
@@ -1392,36 +1392,63 @@ def search(request):
     """
     articles = []
     search_term = None
-
-    if request.POST:
-        search_term = request.POST.get('search')
-        request.session['article_search'] = search_term
-        return redirect(reverse('search'))
-
-    if request.session.get('article_search'):
-        search_term = request.session.get('article_search')
-
-        article_search = submission_models.Article.objects.filter(
-            (Q(title__icontains=search_term) |
-             Q(subtitle__icontains=search_term)) &
-            Q(journal=request.journal)
-        )
-        article_search = [article for article in article_search]
-
-        author_search = search_term.split(' ')
-        from_author = submission_models.FrozenAuthor.objects.filter(
-            (Q(first_name__in=author_search) |
-             Q(last_name__in=author_search)) &
-            Q(article__journal=request.journal)
-        )
-
-        articles_from_author = [author.article for author in from_author]
-        articles = set(article_search + articles_from_author)
+    keyword = None
```

Comment:
`git diff` is still flagging some odd whitespace in this view:

![image](https://user-images.githubusercontent.com/10760678/54918326-db9d8b80-4ef5-11e9-91ae-cccdf327a3bf.png)